igneous/tasks/mesh/multires.py

```diff
@@ -193,12 +193,13 @@ def MultiResShardedMeshMergeTask(
   shard_no:str,
   draco_compression_level:int = 1,
   mesh_dir:Optional[str] = None,
+  frag_path:Optional[str] = None,
   num_lod:int = 1,
   spatial_index_db:Optional[str] = None,
   min_chunk_size:Tuple[int,int,int] = (128,128,128),
   progress:bool = False
 ):
-  cv = CloudVolume(cloudpath, spatial_index_db=spatial_index_db)
+  cv = CloudVolume(cloudpath, spatial_index_db=spatial_index_db, cache=True)
```

Comment:
This should be a flag. Igneous doesn't use disk space by default, which is a major feature that makes configuration easier.